pages/glossary/visible-inner-text.md

```diff
@@ -0,0 +1,38 @@
+# Visible inner text
+
+(The "visible inner text" defined here is similar to, but not the same as, [visible text content][].)
+
+The <dfn>visible inner text of a node</dfn> depends on the kind of node. 
+
+The <dfn id="for-text">visible inner text of a [text node][]</dfn> is:
+-   if the [text node][] is [visible][], its visible inner text is its [data][];
```

Comment:
Should that data be somehow normalized? (contiguous `[whitespace][]` replaced by a single ASCII whitespace, notably).

Reply:
I agree that it should be normalized like that somewhere.   The question is where.  I don't have a strong opinion on whether it gets normalized here, in the definition.  Currently it gets normalized later, in my algorithm, in the step "Split the string into a list of strings, using a whitespace regular expression as the separator".

Reply:
I think it may be worth normalising it here (or maybe only once when computing the visible inner text of elements) since it would make the definition easier to re-use in other rules if need be.

Reply:
Done in https://github.com/act-rules/act-rules.github.io/pull/2075/commits/a141df88c065e845e2bf49ad8a9481b2f21ea371